airbyte-cdk/python/unit_tests/sources/file_based/scenarios/csv_scenarios.py

```diff
@@ -467,48 +467,46 @@
     )
 ).build()
 
-multi_format_analytics_scenario: TestScenario[InMemoryFilesSource] = (
+csv_analytics_scenario: TestScenario[InMemoryFilesSource] = (
```

Comment:
This test used to test analytics for multi-format. However, we don't actually support multi format. Although the analytics come out, the read fails silently, which means that we don't get the state message we expect. 

Instead, we test the analytics separately (for the cases we do support - one format) and don't allow a read to silently fail.